ui/src/components/harvest-games/HarvestGamesPage.js

```diff
@@ -19,6 +19,7 @@ import {
 } from '@chakra-ui/react';
```

Comment:
overall something happened with how u edited this file which is causing the `HarvestGamesLeaderboard` to not be rendering in it's tab. Can you check why your changes are breaking it?

Reply:
<img width="306" alt="image" src="https://user-images.githubusercontent.com/16095743/195845449-d7525233-5075-4ef0-a3d7-ecbb554c96b9.png">